### PR TITLE
RTL

### PR DESCRIPTION
**Pull request by shaunbent**
First pass at adding the Sass needed for RTL support into the sass-tools

- Also cleaned up a few functions
- Added `triple` and `third` math functions

**To Do**
- [x] Tests
- [x] Docs
- [x] Bump

### CODE REVIEW

**Comment by mintuz** [comment]
👍

**Comment by sareh** [comment]
Looks great. 👍

File: lib/settings/_rtl.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+//     #{$margin-left}: $gel-spacing-unit;
+// }
+//
+// @type String
+//
+$padding-left:    'padding-left';
+$padding-right:   'padding-right';
+$margin-left:     'margin-left';
+$margin-right:    'margin-right';
+$border-right:    'border-right';
```
Comment (stefano-belloro) [inline lib/settings/_rtl.scss]: hi @shaunbent, I've noticed $border-left is missing; in RN we also support $border-left-width and $border-right-width.
other that that, it looks good!

Comment (shaunbent) [inline lib/settings/_rtl.scss]: I will add them in and compare with the list in RN now

Comment (stefano-belloro) [inline lib/settings/_rtl.scss]: it's here /responsive-news/tabloid/webapp/static/sass/tools/_rtl.scss

**Pull request merged**